docs/quick-start-docker-cluster.md

```diff
@@ -0,0 +1,165 @@
+# Quick start for docker-cluster
+
+The docker-cluster image is similar to docker-standard, but with the following changes and features:
+
+* Cluster of 4 metrictanks (with 2x replication)
+* Separate Graphite monitoring server
+* The metrictank binary is **not** baked in
+* Loads custom configurations and scripts
+* Supports tags
+* More dashboards are available
+* Kafka input and clustering backend
+* Short chunkspan & numchunks (benefits: easy to trigger mem and mem_and_cass requests, frequent cass saves, notifier messages etc)
+* Prometheus monitoring
+* Jaeger tracing
+* caddy-proxied endpooints/datasources to check with different org-id auth headers
+* tsdb-gw
+
+
+The following programs are required to build and run the docker-cluster image:
+
+* [Docker](https://docs.docker.com/install/)
+* [docker-compose](https://docs.docker.com/compose) >= version 1.6
+* [Go](https://golang.org/doc/install) (needed to build metrictank)
+
+## Getting the repository
+
+Pull down the repository and its dependencies: 
+
+```
+go get github.com/grafana/metrictank/...
+cd $GOPATH/src/github.com/grafana/metrictank
+```
+
+## Build metrictank
+
+Build metrictank and the docker images:
+```
+make
+```
+
+`make` creates the folder `$GOPATH/src/github.com/grafana/metrictank/build`. If you experience a permissions error during `make` it is probably related to this. Try either changing ownership (`chown`) or permissions (`chmod`) on the folder to something your current user is able to read and write, then run `make` again.
```

Comment:
you shouldn't get permissions errors. what may happen if you didn't build prior to launching, docker will create directories as root user, which should be deleted.

Reply:
I would like to add a small note that talks about this